database/src/migrations/release.0.5/restoration.sql

```diff
@@ -372,17 +384,17 @@ COMMENT ON TABLE first_nations IS 'A list of first nations.'
 --
 
 CREATE TABLE funding_source(
-    funding_source_id        integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
-    name                     varchar(100)      NOT NULL,
+    funding_source_id        integer          GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    name                     varchar(100)     NOT NULL,
     description              varchar(250),
-    record_effective_date    date              NOT NULL,
+    record_effective_date    date             NOT NULL,
     record_end_date          date,
-    project_id_optional      boolean           NOT NULL,
-    create_date              timestamptz(6)    DEFAULT now() NOT NULL,
-    create_user              integer           NOT NULL,
-    update_date              timestamptz(6),
-    update_user              integer,
-    revision_count           integer           DEFAULT 0 NOT NULL,
+    project_id_optional      character(10)    NOT NULL,

```

Comment:
Did you mean to change this from `boolean` to `character(10)`? Or should it be `character(1)` like the other fields (priority, is_public, etc)?